spec/sucker_punch_spec.rb

```diff
@@ -11,4 +11,19 @@
       SuckerPunch.logger = nil
     end
   end
+
+  describe '.clear_actors' do
+    it "removes sucker_punch actors from registry" do
+      sucker_punch_actor = {
+        "#{SuckerPunch::NAME_PREFIX}_actor".to_sym => "actor"
+      }
+      other_actor = { other_actor: "other_actor" }
+      actors = sucker_punch_actor.merge(other_actor)
+      allow(Celluloid::Actor).to receive(:all).and_return(actors)
```

Comment:
I'd prefer to do the logic here even if it turns the test in to more of an integration test.